### PR TITLE
Add TransactionLog to capture all asset-transferring tx

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: action/receipt_test.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func TestConvert(t *testing.T) {
 		hash.Hash256b([]byte("Pacific")),
 		hash.Hash256b([]byte("Aleutian")),
 	}
-	log := &Log{"1", topics, []byte("cd07d8a74179e032f030d9244"), 1, hash.ZeroHash256, 1, true, "", ""}
+	log := &Log{"1", topics, []byte("cd07d8a74179e032f030d9244"), 1, hash.ZeroHash256, 1, true, false, "", ""}
```
Comment (yutongp) [inline action/receipt_test.go:26]: strongly suggest add field name here. 
otherwise, very confusing

File: action/receipt_test.go [hunk 2]
```diff
@@ -124,7 +118,7 @@ func TestSerDerLog(t *testing.T) {
...
```
Comment (yutongp) [inline action/receipt_test.go:121]: same here

File: action/receipt_test.go [hunk 3]
```diff
@@ -92,7 +86,7 @@ func TestConvertLog(t *testing.T) {
...
```
Comment (yutongp) [inline action/receipt_test.go:89]: same here

File: action/receipt_test.go (hunk 1 above)
Comment (dustinxie) [reply action/receipt_test.go]: fixed

**Pull request merged**